qiskit_experiments/composite/composite_experiment.py

```diff
@@ -63,6 +66,10 @@ def _add_job_metadata(self, experiment_data, job, **run_options):
         # Add sub-experiment options
         for i in range(self.num_experiments):
             sub_exp = self.component_experiment(i)
+            if sub_exp.run_options != Options() or sub_exp.transpile_options != Options():
+                warnings.warn(
+                    "All sub-experiment options are overridden by composite experiment settings."
```

Comment:
Analysis options are not overridden... :-)

Reply:
Good point, I've updated the warning :)